experimental/framework/tinylicious-client/src/tinyliciousService.ts

```diff
@@ -7,10 +7,21 @@ import {
     IContainer,
     IRuntimeFactory,
 } from "@fluidframework/container-definitions";
-import { IDocumentServiceFactory, IUrlResolver } from "@fluidframework/driver-definitions";
+import { Container } from "@fluidframework/container-loader";
+import {
+    IDocumentServiceFactory,
+    IUrlResolver,
+} from "@fluidframework/driver-definitions";
 import { RouterliciousDocumentServiceFactory } from "@fluidframework/routerlicious-driver";
-import { InsecureTinyliciousTokenProvider, InsecureTinyliciousUrlResolver } from "@fluidframework/tinylicious-driver";
-import { getContainer, IGetContainerService } from "./getContainer";
+import {
+    InsecureTinyliciousTokenProvider,
+    InsecureTinyliciousUrlResolver,
+} from "@fluidframework/tinylicious-driver";
+import { getContainer, IGetContainerService } from "@fluid-experimental/get-container";
+
+export interface ITinyliciousServiceConfig {
+    id: string;
+}
 
 export class TinyliciousService implements IGetContainerService {
```

Comment:
Can we not collapse the `TinyliciousService` and the `FluidTinyliciousInstance` together?

Reply:
Icic, they are separate to support the old code. 

Let's just keep the `get-container` as is. Port the required functionality to `FluidTinyliciousInstance`. And mark the `get-container` pkg as deprecated on the README.

Reply:
Yes TinyliciousClient now consolidates both of their behavior together into one class. get-container's functionality has been moved into a private getContainerCore call.

Added the deprecated comment and the associate ticket for cleanup to its README